### ISSUE TITLE
Exception on start

### ISSUE THREAD

**Issue by praseodym** [angr/angr-management#1]
When I try to launch angr-management, I get the following exception:

```
% python -m angrmanagement
No handlers could be found for logger "claripy"
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/mark/angr/angr-management/angrmanagement/__main__.py", line 22, in <module>
    view.show()
  File "/home/mark/.virtualenvs/angr/local/lib/python2.7/site-packages/enaml/widgets/window.py", line 414, in show
    self.initialize()
  File "/home/mark/.virtualenvs/angr/local/lib/python2.7/site-packages/enaml/widgets/window.py", line 192, in initialize
    super(Window, self).initialize()
  File "/home/mark/.virtualenvs/angr/local/lib/python2.7/site-packages/enaml/widgets/toolkit_object.py", line 160, in initialize
    super(ToolkitObject, self).initialize()
  File "/home/mark/.virtualenvs/angr/local/lib/python2.7/site-packages/enaml/core/declarative.py", line 120, in initialize
    self.initialized()
  File "/home/mark/.virtualenvs/angr/local/lib/python2.7/site-packages/enaml/core/declarative_meta.py", line 68, in declarative_change_handler
    engine.write(owner, change['name'], change)
  File "/home/mark/.virtualenvs/angr/local/lib/python2.7/site-packages/enaml/core/expression_engine.py", line 210, in write
    pair.writer(owner, name, change)
  File "/home/mark/.virtualenvs/angr/local/lib/python2.7/site-packages/enaml/core/standard_handlers.py", line 82, in __call__
    call_func(func, (), {}, scope)
  File "angrmanagement/ui/main.enaml", line 101, in <module>
    if len(inst.workspaces) == 0:
AttributeError: 'NoneType' object has no attribute 'workspaces'
Exception TypeError: "'NoneType' object is not callable" in  ignored
Exception TypeError: "'NoneType' object is not callable" in  ignored
````

I'm using a [development installation](https://github.com/angr/angr#development-install). I suppose I'm missing a step before trying to launch angr-management.

**Comment by zardus** [comment]
@jmgrosen, I seem to be getting this error too, now. It started just on my laptop (during the Defcon talk!) but has now begun to plague my desktop as well. Any ideas?

**Comment by ltfish** [comment]
The correct way to manage your angr is the following:
```
python -m angrmanagement path_to_your_binary
```

For example, `python -m angrmanagement my_cool_binary`, and `my_cool_binary` is the binary you want to load in `angr`.

This is still an issue, though. It should not fail with a weird exception just because of a missing command line argument.

**Comment by praseodym** [comment]
Thanks, that did the trick. I did miss some requirements though; see #2.

**Comment by ltfish** [comment]
Thank you for the pull request!

**Comment by jmgrosen** [comment]
This is because of the dynamic workspaces thing I hacked in very quickly -- I always tested it with a command line argument :)

Fixing now.

**Comment by jmgrosen** [comment]
Should be fixed by 5f44c80. Sorry for the wait -- apparently "now" is almost a week :)
